### ISSUE TITLE
Ironbrows weapon shops has wrong effect (dwarfs)

### ISSUE THREAD

**Issue by floskan** [chadvandy/cbfm_wh3#1733]
https://community.creative-assembly.com/total-war/total-war-warhammer/bugs/1171-dwarf-land-mark-ironbrow-s-weapons-shops?page=1

**Comment by mfvreeland** [comment]
Questionable design, perhaps, but not really a bug.

**Comment by floskan** [comment]
It had a different effect a few patches ago. "Weapon strength: +5% for Dwarf units with hand weapons (local armies)" which makes more sense (to me)

**Comment by mfvreeland** [comment]
It makes more sense to me, too; I'd just want a little more confirmation that the change was unintentional. I'm wasn't going to close this issue, though, because it is odd.

**Comment by Drazhir2** [comment]
> It had a different effect a few patches ago. "Weapon strength: +5% for Dwarf units with hand weapons (local armies)" which makes more sense (to me)

May I ask where you saw that? my git-fu failed me and all I could find is this change that happened in patch 3.0.0:
![image](https://github.com/chadvandy/cbfm_wh3/assets/135981276/6968e05b-1334-4aad-88ef-235e18570e86)

**Comment by allenhu0912** [comment]
I recall seeing that in WH2.

**Comment by floskan** [comment]
Yeah I think Allen is correct, I might have selective memory. But I still think this new effect was an error lost in translation between games or something

**Comment by mfvreeland** [comment]
This was recently brought up on Discord, and there is a growing consensus that although this has been this way in WH3 for a long time, it was a regionwide weapon strength buff in WH2, and that just makes too much more sense not to implement. I am inclined to agree at this point.
